src/main/thrift/parquet.thrift

```diff
@@ -511,6 +511,49 @@ struct ColumnMetaData {
    * This information can be used to determine if all data pages are
    * dictionary encoded for example **/
   13: optional list<PageEncodingStats> encoding_stats;
+
+  /** Byte offset from beginning of file to bloom filter data**/
+  14: optional i64 bloom_filter_offset;
+}
+
+/**
+  * Definition of bloom filter algorithm.
+  */
+enum BloomFilterAlgorithm {
+  /** Block based bloom filter. 
+   * The bloom filter bitset is separated into tiny bucket as tiny bloom 
+   * filter, the high 32 bits hash value is used to select bucket, and 
+   * lower 32 bits hash values are used to set bits in tiny bloom filter.
+   * See “Cache-, Hash- and Space-Efficient Bloom Filters”. Specifically, 
+   * the tiny bloom filter size is 32 bytes. The algorithm also needs 8 odd
+   * SALT values (0x47b6137bU, 0x44974d91U, 0x8824ad5bU, 0xa2b7289dU,
```

Comment:
This doe snot describe enough about the SALT to be able to fully reconstruct the algorithm.